src/Views/listings/view.phtml

```diff
@@ -20,28 +20,35 @@ $title = 'listing';
         <div>
             <h1><?= htmlspecialchars($listing->getListingName()) ?></h1>
         </div>
-        <div><h6><?= $listing->getUser()->getOffice()->getAddress() ?>Office</h6></div>
+        <div><h6><?= $listing->getUser()->getOffice()->getAddress() ?> Office</h6></div>
     </div>
     <div class="view-listing-interact listing-grid-design">
         <?php if($listing->getType()=='sell'): ?>
-        <h4 class="zero-margin"><span><?= $listing->getUser()->getFullName() ?></span> is selling for: </h4>
+        <h4 class="zero-margin"><?= $listing->getUser()->getFullName() ?> is selling for:</h4>
         <h4><span class="emphasise-item-text">£<?= htmlspecialchars($listing->getPrice()) ?></h4>
         <?php elseif($listing->getType()=='exchange'): ?>
-        <h4>Is exchanging for:<span class="emphasise-item-text"><?= htmlspecialchars($listing->getDesiredItem()) ?></span></h4>
+        <h4 class="zero-margin"><?= $listing->getUser()->getFullName() ?> Is exchanging for:</h4>
+        <h4><span class="emphasise-item-text"><?= htmlspecialchars($listing->getDesiredItem()) ?></span></h4>
         <?php elseif($listing->getType()=='giveaway'): ?>
-        <h4>Is giving away</h4>
+        <h4 class="zero-margin"><?= $listing->getUser()->getFullName() ?> would like to give this item away:</h4>
         <?php endif; ?>
         <?php if ($listing->getApprovalStatus() === ApprovalStatus::APPROVED): ?>
-        <button type="submit" class="btn btn-primary w-100 mt-4">Submit</button>
+            <?php if($listing->getType()=='sell'): ?>
```

Comment:
getApprovalStatus() should be used instead of getType(), since getType() only returns the data type of the variable